### PR TITLE
feat(core/mdn-annotation): show engine support indicators 

### PR DESCRIPTION
**Pull request by sidvishnoi**
Closes https://github.com/w3c/respec/issues/2861

### CODE REVIEW

File: src/core/mdn-annotation.js [hunk 1]
```diff
@@ -31,6 +31,14 @@ const MDN_BROWSERS = {
   webview_android: "WebView Android",
 };
 
+const localizationStrings = {
+  en: {
+    inAllEngines: "This feature is in all current engines.",
```
Comment (saschanaz) [inline src/core/mdn-annotation.js:36]: "major" engines, maybe. Servo is an engine too 😄

File: src/core/mdn-annotation.js [hunk 2]
```diff
@@ -31,6 +31,14 @@ const MDN_BROWSERS = {
...
+    inSomeEngines: "This feature is in less than two current engines.",
```
Comment (saschanaz) [inline src/core/mdn-annotation.js:37]: "This feature has poor support"? The number should be shown next anyway.

File: src/core/mdn-annotation.js [hunk 3]
```diff
@@ -188,3 +198,29 @@ function findElements(data) {
...
+    return html`<span title="${l10n.inSomeEngines}">🚫</span>`;
+  }
+}
+
+/**
+ * @param {MdnEntry['engines']} engines
+ * @returns {HTMLParagraphElement|undefined}
+ */
+function getEngineSupport(engines) {
+  if (engines.length === 3) {
```
Comment (saschanaz) [inline src/core/mdn-annotation.js:220]: Why 3 when `MDN_BROWSERS` includes many browsers?

File: src/core/mdn-annotation.js (hunk 1 above)
Comment (sidvishnoi) [reply src/core/mdn-annotation.js:36]: Was copying Bikeshed

File: src/core/mdn-annotation.js (hunk 3 above)
Comment (sidvishnoi) [reply src/core/mdn-annotation.js:220]: "blink", "gecko" and "webkit"

Comment (saschanaz) [reply src/core/mdn-annotation.js:220]: Ah okay, it's from https://github.com/w3c/mdn-spec-links/blob/346c5fac134f99f4bd52920a1be1542f661aede6/.browser-compat-data-process.js#L365.

File: src/core/mdn-annotation.js (hunk 1 above)
Comment (saschanaz) [reply src/core/mdn-annotation.js:36]: Do we need to sync the messages?

Comment (sidvishnoi) [reply src/core/mdn-annotation.js:36]: Nope. We need something for starters though..

File: src/core/mdn-annotation.js (hunk 2 above)
Comment (marcoscaceres) [reply src/core/mdn-annotation.js:37]: s/poor/limited maybe.

File: src/core/mdn-annotation.js (hunk 1 above)
Comment (marcoscaceres) [reply src/core/mdn-annotation.js:36]: Agree... we should start with this set. It's nice that both ReSpec and BikeShed are showing the same boxes/info for now.

Comment (saschanaz) [reply src/core/mdn-annotation.js:36]: I think adding "major" should still be fine, though. EdgeHTML is still in a lot of machines and there are also other minor ones.

Comment (saschanaz) [reply src/core/mdn-annotation.js:36]: Is there a reason you don't want it? 😕

Comment (sidvishnoi) [reply src/core/mdn-annotation.js:36]: Oops. Missed your newer comments before merge @saschanaz. Will improve messages in a followup

File: src/core/mdn-annotation.js (hunk 2 above)
Comment (sidvishnoi) [reply src/core/mdn-annotation.js:37]: > "This feature has poor support"? The number should be shown next anyway.

"This feature has limited support (n engines)"?

File: src/core/mdn-annotation.js (hunk 1 above)
Comment (sidvishnoi) [reply src/core/mdn-annotation.js:36]: "This feature is in all major engines"

Comment (sidvishnoi) [reply src/core/mdn-annotation.js:36]: @saschanaz do you think this message is fine? Open to suggestions.

Comment (saschanaz) [reply src/core/mdn-annotation.js:36]: That looks fine 😁

File: src/core/mdn-annotation.js (hunk 2 above)
Comment (saschanaz) [reply src/core/mdn-annotation.js:37]: The support table should show how its limited, so maybe no need for "n engines" part.

**Pull request merged**